### DISCUSSION TITLE
alpha cluster possible?

### DISCUSSION THREAD

**Discussion by mindon** [visioncortex/vtracer#9] [Ideas]
it seems there's no alpha object extract tech yet.

get a glass ball or cup from a photo... 🤔

**Comment by tyt2y3** [comment]
Yes, there is nothing like that yet, nor it is in the scope of this project (vtracer).

**Comment by mindon** [comment]
it seems possible 😄 
ClearGrasp: 3D Shape Estimation of Transparent Objects for Manipulation
https://arxiv.org/abs/1910.02550
